chart/values.yaml

```diff
@@ -127,3 +127,248 @@ owlsui:
+      # owgw
+      - name: owgwws
+        protocol: TCP
+        port: 15002
+        targetPort: owgwws
+      - name: owgwrest
+        protocol: TCP
+        port: 16002
+        targetPort: owgwrest
+      - name: owgwfileup
+        protocol: TCP
+        port: 16003
+        targetPort: owgwfileup
+      - name: owgwrestint
+        protocol: TCP
+        port: 17002
+        targetPort: owgwrestint
+      # owprov
+      - name: owprovrest
+        protocol: TCP
+        port: 16005
+        targetPort: owprovrest
+      - name: owprovrestint
+        protocol: TCP
+        port: 17005
+        targetPort: owprovrestint
+      # owsec
+      - name: owsecrest
+        protocol: TCP
+        port: 16001
+        targetPort: owsecrest
+      - name: owsecrestint
+        protocol: TCP
+        port: 17001
+        targetPort: owsecrestint
+      # rttys
+      - name: rttysdev
+        protocol: TCP
+        port: 5912
+        targetPort: rttysdev
+      - name: rttysuser
+        protocol: TCP
+        port: 5913
+        targetPort: rttysuser
+      - name: rttysweb
+        protocol: TCP
+        port: 5914
+        targetPort: rttysweb
+
+  containerPorts:
+    # healthcheck
+    - name: health
+      containerPort: 8080
+    # owfms
+    - name: owfmsrest
+      containerPort: 16004
+    - name: owfmsrestint
+      containerPort: 17004
+    # owgw
+    - name: owgwws
+      containerPort: 15002
+    - name: owgwrest
+      containerPort: 16002
+    - name: owgwfileup
+      containerPort: 16003
+    - name: owgwrestint
+      containerPort: 17002
+    # owprov
+    - name: owprovrest
+      containerPort: 16005
+    - name: owprovrestint
+      containerPort: 17005
+    # owsec
+    - name: owsecrest
+      containerPort: 16001
+    - name: owsecrestint
+      containerPort: 17001
+    # rttys
+    - name: rttysdev
+      containerPort: 5912
+    - name: rttysuser
+      containerPort: 5913
+    - name: rttysweb
+      containerPort: 5914
+
+  # NOTE set the listed environment variables in order to make proxy working
+  extraEnvVars: []
+#    - name: SVC_DOMAIN
+#      value: "openwifi-test.svc.cluster.local"
+
+  configuration: |
+    global
+      log stdout format raw local0
+      maxconn 1024
+    defaults
+      log global
+      timeout client 60s
+      timeout connect 60s
+      timeout server 60s
+
+    # healthcheck
+    frontend front_healthcheck
+      bind :8080
+      mode http
+      default_backend back_healthcheck
+    backend back_healthcheck
+      mode http
+      http-after-response set-header Access-Control-Allow-Origin "*"
+      http-after-response set-header Access-Control-Max-Age "31536000"
+      http-request return status 200 content-type "text/plain" string "Pong"
+
+    # owfms
+    frontend front_owfms_rest
+      bind :16004
+      mode tcp
+      default_backend back_owfms_rest
+    backend back_owfms_rest
+      mode tcp
+      server svc_owfms_rest owfms-owfms.${SVC_DOMAIN}:16004
```

Comment:
Think we don't have to use the FQDN here. `owfms-owfms` should resolve just fine to the fms service in the same namespace. This way we can spare needing to pass the `SVC_DOMAIN`.

Reply:
I tried to do that initially, but for some reason HAproxy can't resolve this name
I tried `owfms-owfms` and `owfms-owfms,openwifi-test.svc`, but only full service name seems to work
Let me try that again to make sure

Reply:
Mmh that's weird, for me `owfms-owfms` resolves just fine from other pods in the same namespace.

Reply:
Yes and it even resolves just fine inside container with haproxy using `getent hosts owfms-owfms`, but I guess HAProxy does resoling some other way

Reply:
Tested it - somehow now using short service name works just fine.
Deleted all that wasn't required, updated wlan-testing deploy script too